crates/uv-static/src/env_vars.rs

```diff
@@ -525,4 +525,7 @@ impl EnvVars {
 
     /// Ignore `.env` files when executing `uv run` commands.
     pub const UV_NO_ENV_FILE: &'static str = "UV_NO_ENV_FILE";
+
+    /// Skip writing `uv` cache & installer files to site-packages dist-info.
```

Comment:
"Set to false to skip writing `uv` cache & installer files to site-packages dist-info." ?

Reply:
There are no other env vars written in that style, and I believe the wording makes more sense since the name was swapped from `UV_EXTRA_DIST_INFO` to `UV_NO_EXTRA_DIST_INFO`, with the logic inverted from previously.